docs/user/code_structure.rst

```diff
@@ -409,8 +409,8 @@ following lines in the WEC-Sim input file, where ``i`` is the body number and
     body(i)=bodyClass('<bemData>.h5')
     body(i).geometryFile = '<geomFile>.stl'; 
 
-WEC-Sim bodies may be one of four types\: hydrodynamic, flexible, 
-drag, or nonhydrodynamic. These types represent varying degrees of complexity
+WEC-Sim bodies may be one of three types\: hydrodynamic and rigid, hydrodynamic and flexible, 
+or nonhydrodynamic and rigid (drag body). These types represent varying degrees of complexity
 and require various input parameters and BEM data, detailed in the table below.
```

Comment:
There are some typos here that I will made a commit to resolve

Reply:
Feel free to clarify. There's a few overlapping features when we talk about the types of bodies (rigid vs flex, hydro vs nonhydro)

Reply:
Note the code change showing above is outdated and for some reason GitHub is not labeling it as such

Reply:
That's detailed in this section: https://wec-sim.github.io/WEC-Sim/main/user/code_structure.html#body-blocks. 

The rest of the docs Code Structure section have an identical format, first breaking out each class and then the respective objects. Though any differences here go away once the rigid and flex are fully linked